securesystemslib/_gpg/common.py

```diff
@@ -886,7 +877,7 @@ def parse_signature_packet(
     signature = handler.get_signature_params(data[ptr:])
 
     signature_data = {
-        "keyid": "{}".format(keyid),
```

Comment:
I suppose this could be just `"keyid": keyid,` but I I think as a pure conversion commit this is fine too

Reply:
Yes, this line makes no sense like this, will change to `"keyid": keyid`